include/avif/avif_cxx.h

```diff
@@ -0,0 +1,35 @@
+// Copyright 2023 Google LLC
+// SPDX-License-Identifier: BSD-2-Clause
+
+// This is a header only library.
```

Comment:
This comment is misleading, because this header requires the libavif library. We should delete this comment or rephrase it.

Reply:
You are right, let's delete it: the file name is explicit enough.

Reply:
I guess the comment meant that there is a separate libavif_cxx library that consists of only avif_cxx.h. The reason it confused me is that I view avif_cxx.h as an additional header of the libavif library.